pkg/osbuild/containers_input.go

```diff
@@ -26,35 +27,26 @@ func NewContainersInputForSources(containers []container.Spec) ContainersInput {
 		refs[c.ImageID] = ref
 	}
 
-	return ContainersInput{
-		References: refs,
-		inputCommon: inputCommon{
-			Type:   "org.osbuild.containers",
-			Origin: InputOriginSource,
-		},
-	}
-}
-
-func NewLocalContainersInputForSources(containers []container.Spec) ContainersInput {
-	refs := make(map[string]ContainersInputSourceRef, len(containers))
-	for _, c := range containers {
-		if !c.LocalStorage {
-			// skip containers that are not in the local containers storage
-			continue
-		}
-		ref := ContainersInputSourceRef{
-			Name: c.LocalName,
-		}
-		refs[c.ImageID] = ref
+	var typ string
```

Comment:
This is a super nitpick, but maybe we could name this `sourceType`?

Reply:
Thanks! Yes, I like this much better.